node/lib/cmd/commit.js

```diff
@@ -120,25 +120,13 @@ const doCommit = co.wrap(function *(args) {
     const repo = yield GitUtil.getCurrentRepo();
     const cwd = process.cwd();
 
-    if (!args.no_verify) {
-        try {
-            yield Commit.execSubmodulePrecommitHooks(repo,
```

Comment:
`Commit.execSubmodulePrecommitHooks` is removed, isn't it? If so, could you also remove it at line 163?

Reply:
thanks